### PR TITLE
Fix for relative filenames on Windows systems

### LINKED ISSUE #5: Copying doesn't work if non-absolute filenames supplied
```
speedcopy.copyfile("C:/Temp/asdf", "C:/Temp/fdsa")
```
should work exactly the same as
```
os.chdir("C:/Temp")
speedcopy.copyfile("asdf", "fdsa")
```

### PR DESCRIPTION
**Pull request by rxxg**
Fixes #5

### CODE REVIEW

File: tests/test_speedcopy.py [hunk 1]
```diff
@@ -13,18 +14,36 @@ def teadown_function(function):
...
     f.close()
 
     shutil.copyfile(str(src), str(dst))
 
     assert os.path.isfile(str(dst))
 
 
+def test_copy_rel(tmpdir):
+    cwd = os.getcwd()
+    os.chdir(tmpdir)
```
Comment (antirotor) [inline tests/test_speedcopy.py]: this has to be `os.chdir(str(tmpdir)` as tmpdir in pytest is not str but **LocalPath**

Comment (rxxg) [reply tests/test_speedcopy.py]: OK, done.

Comment (rxxg) [reply tests/test_speedcopy.py]: Looks like there was a genuine test failure on the Linux side.  I don't really have the time to dive into that side of things I'm afraid, I have pushed a speculative fix but I have no access to a Linux machine to test.

Comment (antirotor) [reply tests/test_speedcopy.py]: yep, I am on it. It seems that on linux side it doesn't find `tmpdir` for destination and therefor failing while trying to query filesystem type.

**Comment by antirotor** [comment]
Seems ok to me now, thanks for contribution!

**Comment by rxxg** [comment]
> 
> 
> Seems ok to me now, thanks for contribution!

You're welcome.

**Pull request merged**